homeassistant/components/open_meteo/weather.py

```diff
@@ -43,18 +55,53 @@ def __init__(
         *,
         entry: ConfigEntry,
         coordinator: DataUpdateCoordinator[OpenMeteoForecast],
+        hourly: bool,
     ) -> None:
         """Initialize Open-Meteo weather entity."""
         super().__init__(coordinator=coordinator)
         self._attr_unique_id = entry.entry_id
+        self._config = MappingProxyType(entry.data)
+        self._hourly = hourly
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        name = self._config.get(CONF_NAME)
+        name_appendix = ""
+        if self._hourly:
+            name_appendix = " hourly"
+
+        if name is not None:
+            return f"{name}{name_appendix}"
+
+        return f"{DEFAULT_NAME}{name_appendix}"
 
-        self._attr_device_info = DeviceInfo(
+    @property
+    def unique_id(self) -> str:
+        """Return unique ID."""
+        name_appendix = ""
+        if self._hourly:
+            name_appendix = "-hourly"
+        unique_id = f"{self._attr_unique_id}{name_appendix}"
```

Comment:
We cannot use `_attr_unique_id` here at all.

`_attr_unique_id` makes the `unique_id` property unused.

Reply:
Where can I find that behaviour in the docs? 

Can I still use `entry.entry_id` to generate a unique id? Because I don't think I have another source to generate one